config/production.json

```diff
@@ -45,6 +45,7 @@
 		"manage/security": true,
 		"manage/site-settings/analytics": true,
 		"manage/site-settings/categories": true,
+		"manage/site-settings/disconnect-flow": false,
```

Comment:
Omitting a flag will simply assume it's disabled in a certain environment, so it's probably best to keep it out in that case.